frontend/lib/resolution/resolution-types.cpp

```diff
@@ -1073,6 +1073,52 @@ void CandidatesAndForwardingInfo::stringify(
   }
 }
 
+
+// Note (Daniel): the code for 'overloaded' below comes from cppreference:
```

Comment:
are we "allowed" to have this note in here?

Reply:
I don't see why not, but could you elaborate on your concern?

Reply:
Copyright or license issues with using someone else's code. I just know HPE has some rules about this and I don't know what they are.

But looking at https://en.cppreference.com/w/Cppreference:FAQ, "What can I do with the material on this site?", I think it's fine.